tests/integration_tests/detectionv2_subclass_test.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+
+import super_gradients
+from super_gradients.training.datasets import PascalVOCDetectionDataSetV2
+from super_gradients.training.transforms import DetectionMosaic, DetectionPaddedRescale, DetectionTargetsFormatTransform
+from super_gradients.training.utils.detection_utils import DetectionTargetsFormat
+
+
+class SubclassingIntegrationTest(unittest.TestCase):
+    def setUp(self) -> None:
+        super_gradients.init_trainer()
+        self.BATCH_SIZE = 64
+        self.PASCAL_CLASS_INCLUSION_LISTS = [['aeroplane', 'bicycle'],
+                                             ['bird', 'boat', 'bottle', 'bus'],
+                                             ['pottedplant'],
+                                             ['person']]
+        transforms = [DetectionMosaic(input_dim=(640, 640), prob=0.8),
+                      DetectionPaddedRescale(input_dim=(640, 640), max_targets=120),
+                      DetectionTargetsFormatTransform(output_format=DetectionTargetsFormat.XYXY_LABEL)]
+        self.PASCAL_BASE_CONFIG = dict(data_dir='/home/louis.dupont/data/pascal_unified_coco_format/',
```

Comment:
Again, what's with all these uppercase letters ? Where did you see we do this ?

Reply:
Same, changed in d5bc0d4b6a901e2db163f8fa85597a762e1142eb